User Guide.md

```diff
@@ -1400,54 +1400,54 @@ bash $
 ```
 >Note: such grouping is only possible with labeled values (obviously), it won't be possible to group array elements that easily,
 > e.g., let's break array into pairs:
->```bash
->bash $ array='[0,1,2,3,4,5,6,7,8,9]'
->bash $ <<<$array jtc -w[::2] -w[1::2] -j -tc
->[ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 ]
->bash $ 
->```
-> it won't work even if we try relating walks:
->```bash
->bash $ <<<$array jtc -w[::2] -w'[::2]<I>k[-1]>I<t1' -j -tc
->[ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 ]
->bash $ 
->```
+```bash
+bash $ array='[0,1,2,3,4,5,6,7,8,9]'
+bash $ <<<$array jtc -w[::2] -w[1::2] -j -tc
+[ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 ]
+bash $
+```
+ it won't work even if we try relating walks:
```

Comment:
non-code lines in such quoted blocks should be quoted, this one is missing, i.e.:
`> it won't work even if we try relating walks:`

Reply:
I've changed it back.